tests/requirements.txt

```diff
@@ -4,3 +4,8 @@ pytest>=6.0.0;python_version>="3.0"
 pytest-timeout
 requests-mock
 trivup>=0.8.2
+fastavro
+avro;python_version<="3.0"
+avro-python3;python_version>="3.0"
+jsonschema
+protobuf
```

Comment:
These deps should not be here, they're already on the avro/protobuf/etc sub-packages and those dependencies will be automatically resolved if you install the sub-packages with `pip` rather than `setup.py`.

This requirements file is for the **test code** only